xwiki-platform-core/xwiki-platform-rendering/xwiki-platform-rendering-macros/xwiki-platform-rendering-macro-include/src/test/java/org/xwiki/rendering/internal/macro/IncludeMacroTest.java

```diff
@@ -536,4 +536,46 @@ private XDOM getXDOM(String content) throws Exception
 
         return blocks;
     }
+    
+    @Test
+    void executeIncludeMacroWhenExcludeFirstHeadingTrue() throws Exception 
+    {
+        // Expected content containing no heading.
```

Comment:
Then why are there sections? This looks weird to have a section inside a section for no reason I can see.